packages/language-support/src/tests/autocompletion/procedureCompletion.test.ts

```diff
@@ -1,19 +1,30 @@
 import { CompletionItemKind } from 'vscode-languageserver-types';
 import { DbSchema } from '../../dbSchema';
+import { testData } from '../testData';
 import { testCompletions } from './completionAssertionHelpers';
 
 describe('Procedures auto-completion', () => {
   const dbSchema: DbSchema = {
-    procedureSignatures: {
-      'tx.getMetaData': { label: 'tx.getMetaData' },
-      'jwt.security.requestAccess': { label: 'jwt.security.requestAccess' },
+    procedures: {
+      'tx.getMetaData': { ...testData.emptyProcedure, name: 'tx.getMetaData' },
```

Comment:
There are a few choices to make this better. Either we export empty procedures and functions that we can use when setting up the test data, as I've done here. Or we can export a method instead.

Reply:
I think it makes sense to start like this and then move to a function is we need more functionality later